spec/services/get_application_forms_ready_to_decline_by_default_spec.rb

```diff
@@ -1,33 +1,32 @@
 require 'rails_helper'
 
-RSpec.describe GetApplicationChoicesReadyToDeclineByDefault do
+RSpec.describe GetApplicationFormsReadyToDeclineByDefault do
   around do |example|
     Timecop.freeze(Time.zone.local(2019, 12, 4, 12, 0, 0)) do
       example.run
     end
   end
 
-  let(:application_form) { create :application_form }
-
-  def create_application(status:, decline_by_default_at:)
-    create(
+  def create_application_form(status:, decline_by_default_at:)
+    application_choice = create(
       :application_choice,
-      application_form: application_form,
       status: status,
       decline_by_default_at: decline_by_default_at,
     )
+
+    application_choice.application_form
   end
 
   it 'returns application choices with decline_by_default_at in the past' do
-    application1 = create_application(
+    application1 = create_application_form(
       status: 'offer',
       decline_by_default_at: 1.business_days.ago,
```

Comment:
While we're here, suggest ordering these applications by their decline_by_default_at date - makes it a bit easier to follow when we get to the assertions in this scenario.

Reply:
Refactored to give them meaningful names.